ood-portal-generator/spec/fixtures/ood-portal.conf.all

```diff
@@ -195,6 +195,26 @@ Listen 8080
     LuaHookLog analytics.lua analytics_handler
   </Location>
 
+  # CORS setup for API requests
+  <Location "/pun/test">
```

Comment:
Preflights not withstanding, this Location should still require authentication right?

Reply:
In general yes, but for this test, I manually configured `/pun/test` as the URL for the API, so the OPTIONS requests to it are not authenticated.

Should the API URL here be configurable or just be: `"<%= @pun_uri %>/api"`?

Reply:
Yea I think it should be hard coded to `"<%= @pun_uri %>/dashboard/api"`. That seems ripe for misconfiguration debugging nightmare.

Sorry - back to auth, just to be clear - we will have to lock this down but OPTIONS will have to work without authentication.  That's something we'll have to pick up later?

Reply:
I don't think so, it is already locked down as it is.
All requests to `"<%= @pun_uri %>"` are authenticated apart from OPTIONS requests to`"<%= @pun_uri %>/sys/dashboard/api"` that Apache will return 204 + the CORS headers.

Is it safe to assume that the Dashboard API will be deployed under the URL: `"<%= @pun_uri %>/sys/dashboard/api"`?
As once hardcoded, if the dashboard app is deployed under a different URL, CORS will no longer work.

Reply:
I have updated the default value for origin to empty, so that admins will have to provide a value.

As well, updated the default value for the CORS path to default to "<%= @pun_uri %>/sys/dashboard/api"